_includes/scripts.html

```diff
@@ -8,7 +8,7 @@
   {% assign _src = _script.src | default: _script %}
 <script src="{{ _src | relative_url }}"></script>
 {% endfor %}{% endfor %}
-{% if page.layout == "help" and page.category != "verify-your-identity" %}
-  <script src="https://touchpoints.app.cloud.gov/touchpoints/58f51d4d.js"></script>
+{% if layout.include_touchpoints and page.category != "verify-your-identity" %}
   <script src="{{'/assets/js/touchpoints_translations.js' | relative_url}}"></script>
-{% endif %}  
+  <script src="https://touchpoints.app.cloud.gov/touchpoints/58f51d4d.js" async onload="translateTouchpoints()"></script>
```

Comment:
this approach relies on this `onload` being hardcoded here, which is not super common, and easy to copy-paste over with updated JS, so I think that makes it brittle.

another approach would be to add the `onload` dynamically in JS, similar to https://cleverbeagle.com/blog/articles/tutorial-how-to-load-third-party-scripts-dynamically-in-javascript

We could even load the touchpoints JS from our own `touchpoints_translations` but I think having it here lets the browsers cache/fetch as early as possible